ros/angel_system_nodes/angel_system_nodes/activity_classification/activity_classifier_tcn.py

```diff
@@ -762,7 +781,7 @@ def _process_window(self, window: InputWindow) -> ActivityDetection:
 
         return activity_msg
 
-    def _collect_results(self, msg: ActivityDetection):
+    def _collect_image(self, end_frame_time) -> int:
```

Comment:
missing time type annotation
```suggestion
    def _collect_image(self, end_frame_time: Time) -> int:
```